From: "Barak A. Pearlmutter" <barak+git@cs.nuim.ie>
Date: Tue, 10 Dec 2013 10:31:14 +0000
Subject: makefile clean version.txi

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index c2289d3..0c2db13 100644
--- a/Makefile
+++ b/Makefile
@@ -304,7 +304,7 @@ clean:
 distclean:	realclean
 realclean:
 	-rm -f *~ *.bak *.orig *.rej TAGS core a.out *.o \#*
-	-rm -f jacal.info*
+	-rm -f jacal*.info* version.txi
 	-rm -rf *temp jacal.html JACAL-*.exe
 realempty:	temp/jacal
 	-rm -f $(allfiles)
