From: "Barak A. Pearlmutter" <barak+git@cs.nuim.ie>
Date: Tue, 10 Dec 2013 10:32:06 +0000
Subject: guile version

---
 jacal.sh | 9 +++++----
 1 file changed, 5 insertions(+), 4 deletions(-)

diff --git a/jacal.sh b/jacal.sh
index 9fa1ec5..ac3cc62 100755
--- a/jacal.sh
+++ b/jacal.sh
@@ -1,4 +1,3 @@
-
 ##"jacal" sh script; Find a Scheme implementation and run JACAL in it.
 # Copyright (C) 2001, 2003, 2004 Aubrey Jaffer
 #
@@ -26,8 +25,9 @@ Usage: jacal SCHEME
 
 Usage: jacal
 
-  Run JACAL using (MIT) 'scheme', 'scm', 'gsi', 'mzscheme', 'guile',
+  Run JACAL using (MIT) 'scheme', 'scm', 'gsi', 'mzscheme', 'guile-1.6',
   'slib48', 'larceny', 'scmlit', 'elk', 'sisc', or 'kawa'."
+# in preceding notice, change guile to guile-1.6, by tb.
 
 case "$1" in
     -v | --ver*) echo jacal "$VERSION"; exit 0;;
@@ -46,8 +46,9 @@ if [ -z "$command" ]; then
 	command=gsi; implementation=gam
     elif type mzscheme>/dev/null 2>&1; then
 	command=mzscheme; implementation=plt
-    elif type guile>/dev/null 2>&1; then
-	command=guile; implementation=gui
+# Changed from looking for 'guile' to 'guile-1.6' by tb.
+    elif type guile-1.6>/dev/null 2>&1; then
+	command=guile-1.6; implementation=gui
     elif type slib48>/dev/null 2>&1; then
 	echo "do 'cd ${JACALDIR}; make jacal48'"
     elif type scheme48>/dev/null 2>&1; then
